packages/material-ui-lab/src/TreeItem/TreeItem.test.js

```diff
@@ -32,6 +33,10 @@ describe('<TreeItem />', () => {
   }));
 
   describe('warnings', () => {
+    beforeEach(() => {
+      PropTypes.resetWarningCache();
```

Comment:
Cleanup of #21695 which added a test that would fail in in watchmode on subsequent runs.